core-scroll-header-panel.css

```diff
@@ -26,6 +30,11 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   overflow-y: auto;
 }
 
+:host([fullWindow]) #mainContainer {
+  position: relative;
+  width: calc(100% - {{!drawerPanel.narrow && drawerPanel.drawerWidth}});
```

Comment:
Yes,  it works with native shadow dom. It doesn't work under the polyfill (other browsers besides chrome). See https://github.com/Polymer/polymer/issues/270#issuecomment-52258823